var/spack/repos/builtin/packages/alluxio/package.py

```diff
@@ -19,6 +19,7 @@ class Alluxio(Package):
 
     license("CC0-1.0")
 
+    version("3.0.9", sha256="50e031ebc4de257f5676eb8b33029c3017a2c7d6864a0f5fbc68210963a8c3e1")
```

Comment:
I appear to not be able to confirm this checksum.  Getting `Error: Could not find any remote versions for alluxio`.

Reply:
Ah! Good find! When prototyping my updated script, I'd thought the script incorrectly parsed the version as 309 instead of 3.0.9, but no the authors changed the version format to now be 308, 309, etc...

Reply:
FYI. Looks like `alluxio@2.2.0` may not checksum anymore.

Reply:
Nvm.  Might need to update my helper script.